evap/contributor/views.py

```diff
@@ -29,7 +29,7 @@ def index(request):
             Q(evaluations__contributions__contributor=user)
         )
     )
-    own_evaluations = [evaluation for course in own_courses for evaluation in course.evaluations.all() if evaluation.state != "new" and evaluation.can_user_see_evaluation(user)]
+    own_evaluations = [evaluation for course in own_courses for evaluation in course.evaluations.all() if evaluation.state in evaluation.can_user_see_evaluation(user)]
```

Comment:
Quick comment for others: This is broken here but fixed again in 5b7940c, another fixup commit.

Reply:
sorry :(